addons/sourcemod/scripting/rngfix.sp

```diff
@@ -676,8 +585,6 @@ void RunPreTickChecks(int client, Handle hParams)
 
 	// These roughly replicate the behavior of their equivalent CGameMovement functions.
 
```

Comment:
Are you sure that all of this stuff is related to csgo only ?

Reply:
```
// In CSS and CSGO but apparently not used in CSS
ConVar g_cvTimeBetweenDucks;

// CSGO-only
ConVar g_cvJumpImpulse;
ConVar g_cvAutoBunnyHopping;
```

Based on these comment, I changed the code, but I would like to have your point of view.